src/pages/workspaces/workspace/Tools.js

```diff
@@ -183,13 +297,18 @@ export const Tools = _.flow(
         })
       ]),
       div({ style: styles.cardContainer(listView) }, [
-        _.map(config => {
-          return h(ToolCard, {
-            onCopy: () => this.setState({ copyingTool: { namespace: config.namespace, name: config.name } }),
-            onDelete: () => this.setState({ deletingTool: { namespace: config.namespace, name: config.name } }),
-            key: `${config.namespace}/${config.name}`, namespace, name, config, listView
-          })
-        }, configs),
+        h(Clickable, {
+          style: { ...styles.card, ...styles.shortCard, color: colors.green[0], fontSize: 18, lineHeight: '22px' },
+          onClick: () => this.setState({ findingTool: true })
+        }, [
+          'Find a Tool',
+          icon('plus-circle', { size: 32 })
```

Comment:
This plus circle is much larger than any others in Terra, the rest are `size: 21` aside from "Create a New Workspace". I recommend 21 here.